### ISSUE TITLE
Fallback tag patterns do not work

### ISSUE THREAD

**Issue by dpitch40** [quodlibet/quodlibet#4417]
Steps to reproduce
------------------
According to the [tag pattern docs](https://quodlibet.readthedocs.io/en/latest/guide/tags/patterns.html), these two tag patterns should be equivalent:
- `<<albumartist>||<artist>>`
- `<albumartist|<albumartist>|<artist>>`

Expected Output
---------------
When used as a custom header, both of these tag patterns should output the albumartist tag if it exists and fall back to the artist tag if not.

Actual Output
-------------
The second pattern works as expected, but the first one outputs nothing.

Test System
-----------

### Which version of Quod Libet?

4.6.0 (also observed on 4.4.0)

### Which operating system 
Linux Mint 21.2

Additional Information
----------------------
The first tag pattern not working is surprising since I copied it directly from the documentation.

![QuodLibet](https://github.com/quodlibet/quodlibet/assets/1520538/fba0213b-36e0-468c-abb8-fac4d76b431f)

**Comment by afontenot** [comment]
This feature wasn't added until after the 4.6.0 release, it'll be in the next release.

It's not clear to me how new documentation gets pushed to Read the Docs. Oddly, there are not any versions listed after "4.0", so many users may end up looking at `latest` instead. Is this something that could be fixed so we have properly versioned documentation, @declension?

**Comment by dpitch40** [comment]
Funny, I have had the exact same problem at my work. I see "latest" is the only version available

**Comment by declension** [comment]
Yes, we used to have this IIRC, but some years ago after a RTD change (on our end perhaps?) we stopped, and nobody seemed to care. I would quite like it back. 

In the meantime the fix here is (sorry I forgot to add this in the review @afontenot) probably just to mention a version in the docs. We've stripped most of the previous ones of these out now (after they're been released for ages) but there are a [few examples of this in the current docs](https://quodlibet.readthedocs.io/en/latest/guide/searching.html#searching-a-specific-tag).
